src/sentry/api/serializers/models/organization.py

```diff
@@ -291,12 +291,15 @@ def get_feature_set(
                     # Remove the organization scope prefix
                     feature_set.add(feature_name[len(_ORGANIZATION_SCOPE_PREFIX) :])
 
-        # Do not include the onboarding feature if OrganizationOptions exist
-        if (
-            "onboarding" in feature_set
-            and OrganizationOption.objects.filter(organization=obj).exists()
-        ):
-            feature_set.remove("onboarding")
+        if "onboarding" in feature_set:
+            all_required_onboarding_tasks_complete = OrganizationOption.objects.filter(
+                organization_id=obj.id, key="onboarding:complete"
+            ).exists()
+
+            # Do not include the onboarding feature if all required onboarding tasks are completed
+            # The required tasks are defined in https://github.com/getsentry/sentry/blob/master/src/sentry/models/organizationonboardingtask.py#L147
```

Comment:
This reference may drift on master, it may be more useful to point to the permalink so someone can find this list based on the changes to the file: https://github.com/getsentry/sentry/blob/797e317dadcec25b0426851c6b29c0e1d2d0c3c2/src/sentry/models/organizationonboardingtask.py#L147

Reply:
ok... but it will contain hashes